versioned_docs/version-8.2/reference/alpha-features.md

```diff
@@ -5,43 +5,49 @@ sidebar_label: Alpha features
 description: "Use alpha features to learn about upcoming changes, try them out, and share feedback."
 ---
 
-Use alpha features to learn about upcoming changes, try them out, and share feedback.
+You can use alpha features to learn about upcoming changes, try them out, and share feedback.
 
-### Alpha
+:::info
+To understand the difference between an alpha feature and an alpha release, see [alpha features and releases](/docs/reference/release-policy.md#alpha-features-and-releases).
```

Comment:
```suggestion
To understand the difference between an alpha feature and an alpha release, see [alpha features and releases](/reference/release-policy.md#alpha-features-and-releases.md).
```

This is blocking and need to be corrected in all versions! Using `/docs/` will route to `/next/` and not respect the version selected in the drop down. For example, clicking this link in 8.2 takes you to `/next/` (aka 8.6).

Reply:
@akeller Great spot thank you - I've changed all these links across the versions to use the correct syntax 👍